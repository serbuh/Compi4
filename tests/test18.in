// Simple while statement
int i = 0;

while(i < 4) {
  print "This should appear 4 times";
  i = i + 1;
}
print "End of while";
