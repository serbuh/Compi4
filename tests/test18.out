This should appear 4 times
This should appear 4 times
This should appear 4 times
This should appear 4 times
End of while
